### DISCUSSION TITLE
Should Ratatui support `NO_COLOR` out of the box?

### DISCUSSION THREAD

**Discussion by kdheepak** [ratatui/ratatui#878] [Ideas]
https://no-color.org/

**Comment by orhun** [comment]
Yup, I think we should support `NO_COLOR` environment variable. It is supported by default by many CLI apps/libs.

**Comment by Valentin271** [comment]
Yep I agree too, plus it shouldn't be that hard.

**Comment by kdheepak** [comment]
I opened an issue for it!

**Comment by Valentin271** [comment]
Issue link: https://github.com/ratatui-org/ratatui/issues/884. This can probably be closed then, subsequent discussions will take place in the issue.
